src/web/mjs/connectors/mavimanga.mjs

```diff
@@ -0,0 +1,61 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MaviManga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mavimanga';
+        super.label = 'Mavi Manga';
+        this.tags = [ 'manga', 'turkish' ];
+        this.url = 'https://mavimanga.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'span.mangasc-title');
+        let id = uri.pathname;
+        let title = data[0].textContent;
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(this.url + '/manga-listesi/', this.requestOptions);
+        let pages = await this.fetchDOM(request, 'div.navigation li:nth-last-child > a');
+        pages = Number(pages[0].pathname.match(/[\w\W]+\/(\d+)\/$/)[1]);
+
+        let data;
```

Comment:
Can be declared within the loop, there is no performance gain in reusing the same variable